python/src/aiconfig/eval/lib.py

```diff
@@ -195,8 +195,8 @@ async def _run(input_datum: str) -> Result[TextOutput, str]:
     res_outputs_: list[Result[TextOutput, str]] = []
     for input_datum in all_inputs:
         res_outputs_.append(await _run(input_datum))
-    res_outputs = cu.result_reduce_list_all_ok(res_outputs_)
-    # res_outputs = await cu.result_reduce_list_all_ok_async(list(map(_run, all_inputs)))
+    res_outputs = core_utils.result_reduce_list_all_ok(res_outputs_)
+    # res_outputs = await core_utils.result_reduce_list_all_ok_async(list(map(_run, all_inputs)))
```

Comment:
delete?

Reply:
this is to remind me to do concurrent async calls once supported.